src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageItemListViewModel.cs

```diff
@@ -427,7 +427,7 @@ public void UpdatePackageStatus(IEnumerable<PackageCollectionItem> installedPack
             // Get the minimum version installed in any target project/solution
             InstalledVersion = installedPackages
                 .GetPackageVersions(Id)
-                .MinOrDefault();
+                .MaxOrDefault();
```

Comment:
Does this have any unintended consequences? I'm thinking some logic may rely on the Min. Perhaps Consolidate tab when determining the lowest version installed that should be updated to match another project?

Reply:
It has some usage apart from displaying the installed version. The diagram below outlines its usage.  

![image](https://user-images.githubusercontent.com/1192347/81332491-04300880-9058-11ea-86fe-537be1d0e301.png)

Reply:
Right,  I'm pretty sure the DetailControlModel logic will change with this PR. I think it will cause the Installed Tab to show a different version in the Details Pane Version dropdown list for the selected package (see screenshot). 

Currently, it selects the same version you have installed version on this tab (plus other logic). Could you verify this behavior using your build? Happy to discuss further.

Btw... perhaps it's not a problem if we switch that dropdown to the highest version on this tab, but we should discuss with PM.

Please comment with how this behavior is affected:
![image](https://user-images.githubusercontent.com/49205731/81421825-18d7d380-9120-11ea-85b2-ade597419719.png)

Reply:
It only changes the version in the Consolidate tab. See list and images below.

- Browse: No
- Installed: No
- Updates: No
- Consolidate: Yes

Scenario: A solution with 2 projects with the following packages installed):

- ConsoleFW1: (text2xml.lib 1.0.0, polly 6.1.0)
- ConsoleFW2: (text2xml.lib 1.1.1)

### Before

Browse

![image](https://user-images.githubusercontent.com/1192347/81446436-bbe41980-912f-11ea-9fe8-5fe3b62b5e3e.png)

Installed

![image](https://user-images.githubusercontent.com/1192347/81446483-d3230700-912f-11ea-9a6d-e9d6ea6d434a.png)

Updates

![image](https://user-images.githubusercontent.com/1192347/81446505-e59d4080-912f-11ea-9d25-caeb9eeb5c47.png)

Consolidate

![image](https://user-images.githubusercontent.com/1192347/81446539-f8b01080-912f-11ea-927f-e96f91826966.png)


### After

Browse

![image](https://user-images.githubusercontent.com/1192347/81445691-3875f880-912e-11ea-9b97-713c1d9a646f.png)

Installed

![image](https://user-images.githubusercontent.com/1192347/81445739-53486d00-912e-11ea-959e-7f01d3497bbf.png)

Updates

![image](https://user-images.githubusercontent.com/1192347/81445764-64917980-912e-11ea-8d34-91e681f01f4f.png)

Consolidate

![image](https://user-images.githubusercontent.com/1192347/81445814-7d9a2a80-912e-11ea-8305-d681ee7346b8.png)